### PR TITLE
[viviuds] Rename fail-fast properties to avoid bdd reference

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: docs/modules/ROOT/pages/tests-configuration.adoc [hunk 1]
```diff
@@ -68,15 +68,15 @@ NOTE: The properties marked with *bold* are mandatory.
 |`PT3H`
 |The max duration of the single story in the batch.
 
-|`bdd.batch.fail-fast`
+|`engine.batch.fail-fast`
```
Comment (valfirst) [inline docs/modules/ROOT/pages/tests-configuration.adoc:71]:
```suggestion
|`batch.fail-fast`
```

File: docs/modules/ROOT/pages/tests-configuration.adoc [hunk 2]
```diff
@@ -68,15 +68,15 @@ NOTE: The properties marked with *bold* are mandatory.
...
 |`false`
 |If set to `true` the subsequent batches execution will be stopped after first failed assertion
 
-|`bdd.story.fail-fast`
+|`engine.story.fail-fast`
```
Comment (valfirst) [inline docs/modules/ROOT/pages/tests-configuration.adoc:75]:
```suggestion
|`story.fail-fast`
```

File: docs/modules/ROOT/pages/tests-configuration.adoc [hunk 3]
```diff
@@ -68,15 +68,15 @@ NOTE: The properties marked with *bold* are mandatory.
...
 |`false`
 |If set to `true` the story execution will be stopped after first failed assertion
 
-|`bdd.scenario.fail-fast`
+|`engine.scenario.fail-fast`
```
Comment (valfirst) [inline docs/modules/ROOT/pages/tests-configuration.adoc:79]:
```suggestion
|`scenario.fail-fast`
```

**Review by valfirst — CHANGES_REQUESTED** [review]
Also deprecated properties mapping is missing

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**